### PR TITLE
Feature/updatenickname

### LINKED ISSUE #267: [Feature] Nickname in user personal information field
### Is this a feature for the SudoSOS point of sale or for the SudoSOS dashboard?

Dashboard

### What would you like?

A "nickname" field in the card to change a user's personal information, as we do support this functionality in the backend.

### Why is this needed?

_No response_

### Other information

_No response_

### PR DESCRIPTION
**Pull request by noelvandevrie**
# Description
Pull Request addresses the problem that nicknames cannot be changed in the frontend.
Pull Request adds this functionality to the individual user overview in the admin panel.
before
after

## Related issues/external references
#267

## Types of changes
- New feature _(non-breaking change which adds functionality)_
- Style _(Change that do not affect the functionality of the code)_

### CODE REVIEW

**Review by JustSamuel — COMMENTED** [review]
Try and follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) in the future, as we use them to create changelogs and versions.

File: apps/dashboard/src/locales/en.json [hunk 1]
```diff
@@ -298,6 +298,7 @@
     "Email address": "Email address",
     "First name": "First name",
     "Last name": "Last name",
+    "nickname": "Nickname",
```
Comment (JustSamuel) [inline apps/dashboard/src/locales/en.json]: `Nickname`

**Review by JustSamuel — APPROVED** [review]
NIT

File: apps/dashboard/src/locales/en.json (hunk 1 above)
Comment (noelvandevrie) [reply apps/dashboard/src/locales/en.json]: changed in https://github.com/GEWIS/sudosos-frontend/pull/287/commits/395268f8a4f67c96119b5698dc24bc12a2471960

**Pull request merged**